Doc: Update storage volumes > Create an instance (#15469)
canonical/lxd: doc/howto/storage_move_volume.md

(storage-move-instance)=
## Move instance storage volumes to another pool

To move an instance storage volume to another storage pool, {ref}`stop the instance <instances-manage-stop>` that contains the storage volume you want to move.

`````{tabs}
````{group-tab} CLI
Use the following command to move the instance to a different pool:

    lxc move <instance_name> --storage <target_pool_name>
````

````{group-tab} UI

Navigate to the overview page of the selected instance, and click on the {guilabel}`Move` button in the top right corner.

```{figure} /images/instances/instance_overview_page.png
:width: 80%
:alt: LXD Instance overview page
```

> Within the move modal, click {guilabel}`Move instance root storage to a different pool` method to view available storage pools to move to.

Reviewer: ```suggestion
Within the move modal, click {guilabel}`Move instance root storage to a different pool` to view available storage pools to move to.
```
I think without this word this reads nicer.

Reviewer 2: I agree with this. It's either that or add "the" after the word "click", but just removing the word "method" is more concise.

```{figure} /images/instances/move_instance_modal.png
:width: 80%
:alt: LXD Instance root storage move method modal
```
Click {guilabel}`Select` in the row of the target storage pool for the move.

```{figure} /images/instances/move_instance_modal_2.png
:width: 80%
:alt: LXD Instance root storage move pool selection modal
```

> On the resulting confirmation modal, click {guilabel}`Move` to complete the process.

Reviewer: "complete the process" is only true as for the user interaction, the actual move process might take a moment to complete.

Author: I'll say "finalize the process". With American spelling, lol.

## Move instance storage volumes to another pool

To move an instance storage volume to another storage pool, make sure the instance is stopped.

`````{tabs}
````{group-tab} CLI
Then use the following command to move the instance to a different pool:

    lxc move <instance_name> --storage <target_pool_name>
````

````{group-tab} UI

Navigate to the overview page of the selected instance, and click on the {guilabel}`Migrate` button in the top right corner.

> ```{figure} /images/storage/storage_volumes_migrate_1.png

Reviewer 2: Note: I see a different module when the Migrate button is clicked. It says "Choose a migration method" and gives the options "Migrate instance root storage to a different pool" and "Migrate instance to a different project". UI version 6.3-ui-0.16.

Author: Thanks for this spot, Minae! I think I must have got this Migration / copy part confused because the instructions I have provided are indeed for copying **Custom Storage Volumes** not Instance Root Storage.

As I have an upcoming PR on moving and copying storage volumes, I think I'll incorporate that one into this one and expand this a bit by having one commit for the create instances docs and another for moving/copying storage.

:width: 80%
:alt: Add a network to an instance in LXD
```

> Within the migration modal, you can select an alternative storage pool to migrate the module to. Once decided, click the {guilabel}`Select` button on the row of the desired pool.

Reviewer 2: ```suggestion
Within the migration modal, select the target storage pool for the migration.
```

```{figure} /images/storage/storage_volumes_migrate_2.png
:width: 80%
:alt: Add a network to an instance in LXD
```

> A confirmation modal will appear. Click {guilabel}`Migrate` to complete the process.

Reviewer 2: ```suggestion
On the resulting confirmation modal, click {guilabel}`Migrate` to complete the process.
```
